pkg/agent/plugin/svidstore/util.go

```diff
@@ -9,17 +9,11 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
-// TODO: refactor it.
-// It expects selectors like:
-//
-// - aws:name:somename
-// - aws:arn:asd123
-// - aws:kmsid:asd123
-// - gcloud:name:somename
-func ParseSelectors(selectors []*common.Selector, typ string) map[string]string {
+// ParseSelectors parses selectors for SVIDStore plugins
+func ParseSelectors(selectors []*common.Selector) map[string]string {
 	data := make(map[string]string)
 	for _, s := range selectors {
-		if s.Type != typ {
+		if s.Type != strings.ToLower(Type) {
```

Comment:
why always to lower? that is a constant that is lower

Reply:
The constant value is "SVIDStore: https://github.com/MarcosDY/spire/blob/agentless_proposal/pkg/agent/plugin/svidstore/svidstore.go#L24

Reply:
then our selectors will looks like: svidstore:secretname:SOMENAME? I was thinking it will be store